Makefile.gh

```diff
@@ -0,0 +1,14 @@
+# This Makefile contains targets used by GitHub Actions
+
+all:
+	@echo
+	@echo "This file contains targets used by GitHub Actions."
+	@echo
+
+codespell:
+	pip install codespell
```

Comment:
Have you considered adding codespell and bandit to our `requirements-dev.txt`?

Reply:
I got it. The only "minor problem" I see with this approach is this _might_ be installing using a different method of what the developer choose depending on the user/env installing it. It is not respecting `PYTHON_DEVELOP_ARGS` the same way other targets in our main Makefile are.

So if I do the following commands:

```
$ make requirements-plugins
$ make develop
$ make -f Makefile.gh codespell
```

The first  two commands are using the `--user` option if available while the last is not, this _might_ result in different requirements being installed in different places. Maybe not a huge problem, but some might get confused. I'm also not saying that we need to fix this here in this PR, just saying out of loud so we can keep that in mind.

I will create another issue for tracking this.

Reply:
I know. But still relevant the location of the requirements.